src/main/java/com/google/sps/data/Image.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
```

Comment:
I'm not sure what the guidelines say here, but shouldn't there be a Copyright statement at the top of every file?

Reply:
Um but we didn't copy this file from anyone........?

Reply:
Oh yes.
I forgot to add those.

Reply:
Done!